src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -681,10 +770,16 @@ private void DeleteFile(string path)
 
         private void ValidateFreeSpaceForSnapshot(string filePath)
         {
+            if (_directUploadDestination != DirectUploadDestination.Disabled)
+            {
```

Comment:
do you need (or have a way) to validate for free space on the remote destination?

Reply:
The reason for this validation is to make sure that we have enough space on the local machine (for the temp file).

Reply:
and we don't / can't check if we have enough space on the S3 bucket?

Reply:
there is no limit to the amount you can upload to S3 (only a limit of `5TB` for a singe file).